### ISSUE TITLE
Github bookmark error

### ISSUE THREAD

**Issue by Jogai** [daledavies/jump#15]
This bookmark:

    {
        "name": "Github",
        "url" : "https://github.com/notifications?query=is%3Aunread"
    },

Results in this error:

```
GET http://start.ho.me/api/icon.php?siteurl=https://github.com/notifications?query=is%3Aunread 500 (Internal Server Error)
VM33:1 Uncaught (in promise) SyntaxError: Unexpected end of JSON input
```

**Comment by daledavies** [comment]
I've pushed a patch that will fix this, will be available in the v1.1.2 release :)

**Comment by daledavies** [comment]
Release v1.1.2 has been created, images on docker hub etc, so closing this issue. Thank you for raising.
